ratis-test/src/test/java/org/apache/ratis/datastream/TestDataStream.java

```diff
@@ -137,42 +139,30 @@ public void testDataStream(){
   public void runTestDataStream(){
     final int bufferSize = 1024*1024;
     final int bufferNum = 10;
-    final DataStreamOutput out = client.stream();
```

Comment:
We should not remove this test.  This is testing DataStreamOutput which support zero buffering copying.

If there is a need, we should add a separated test for the other cases.